### PR TITLE
Add SSR content

### PR DESCRIPTION
**Pull request by rbrander**

### CODE REVIEW

File: src/content/introduction.md [hunk 1]
```diff
@@ -72,13 +72,61 @@ npm install
...
+- Serve up cached markup on request
+
++++
+
+## Live coding exercise
+
+Code along or just watch
+- Clone the repo: `git clone git@github.com:turbobeast/advanced-react-training.git`
+- Change directory: `cd advanced-react-training`
+- Install dependencies: `npm install`
```
Comment (turbobeast) [inline src/content/introduction.md]: I added `clone`, `cd` and `npm install` to slide `/#/3` "Get Started". Maybe just add `git checkout branch`

Comment (turbobeast) [reply src/content/introduction.md]: Keep `npm install` if there are new deps you don't add while live-coding.

**Comment by rbrander** [comment]
Updated the slide based on your feedback

File: src/content/introduction.md [hunk 2]
```diff
@@ -72,13 +72,64 @@ npm install
...
+
+## Live coding exercise
+
+If you don't have the repo cloned yet:
+- Clone the repo: `git clone git@github.com:turbobeast/advanced-react-training.git`
+- Change directory: `cd advanced-react-training`
+- Install dependencies: `npm install`
+
+Once you have the repo installed:
+- Create a branch off master: `git checkout master -b add-ssr`
```
Comment (turbobeast) [inline src/content/introduction.md:123]: it should be `git checkout -b add-ssr`

File: src/content/introduction.md [hunk 3]
```diff
@@ -72,13 +72,64 @@ npm install
...
+- What is SSR?
+- Pros/Cons
+- High-level implementation overview
+
++++
+
+## What is SSR?
+- Server-Side Rendering
+- AKA Universal, Isomorphic
+- initial DOM and state are generated and served
```
Comment (turbobeast) [inline src/content/introduction.md]: capitalize `Initial` for consistency

**Review by turbobeast — CHANGES_REQUESTED** [review]
looks good, just needs two small changes.

File: src/content/introduction.md (hunk 2 above)
Comment (rbrander) [reply src/content/introduction.md:123]: I intentionally did it this way to ensure they would be branching off from the starting point.  In the event they were already on a branch for the routing, using your command would actually branch off the current branch, and not master.

Comment (turbobeast) [reply src/content/introduction.md:123]: okay, that makes sense

**Review by turbobeast — APPROVED** [review]
looks good :shipit:

**Pull request merged**